scripts/exrap_dump.sh

```diff
@@ -1106,7 +1107,7 @@ else
 
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_7} 1 1bhrs4 \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_7} 1 1bhrs4 \
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 1bhrs4 \

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_7} 1 1bhrs4 \